### PR TITLE
Added recipe for Sequenza

### PR DESCRIPTION
**Pull request by micknudsen**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-sequenza/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - r-base
+    - r-squash
+
+test:
+  commands:
+    - $R -e "library('sequenza')"           # [not win]
+    - "\"%R%\" -e \"library('sequenza')\""  # [win]
+
+about:
+  home: 'http://cbs.dtu.dk/biotools/sequenza/, Mailing list: https://groups.google.com/forum/#!forum/sequenza-user-group'
```
Comment (bgruening) [inline recipes/r-sequenza/meta.yaml]: please only the homepage

File: recipes/r-sequenza/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  license: GPL-3
+  summary: Tools to analyze genomic sequencing data from paired normal-tumor samples, including
+    cellularity and ploidy estimation; mutation and copy number (allele-specific and
+    total copy number) detection, quantification  and visualization.
+  license_family: GPL3
```
Comment (bgruening) [inline recipes/r-sequenza/meta.yaml]: please add license_file and if you have a doi in extras/identifiers

Comment (micknudsen) [reply recipes/r-sequenza/meta.yaml]: Does this recipe need more before it can be merged?

**Comment by micknudsen** [comment]
Does this recipe need more work before it can be merged?

**Comment by micknudsen** [comment]
Any chance that someone could find the time to merge this? @johanneskoester? Thanks!

**Comment by bgruening** [comment]
Urgs, sorry. Please feel free to ping me earlier. Sorry that this slipped through!

**Comment by micknudsen** [comment]
@bgruening Thanks! I try to resist the urge to ping people too often. Don't wan't to come across as rude 😄

**Comment by bgruening** [comment]
Michael, very much appreciated, but this was unacceptable late :)
https://anaconda.org/bioconda/r-sequenza

**Pull request merged**